content/en/pr_gates/_index.md

```diff
@@ -50,87 +45,61 @@ Use PR Gates to:
 
 You can configure PR Gates rules for the following categories: 
 
-[Test Optimization][9]
-
-: <br> - New flaky tests <br> - Code coverage
-
-[Static Analysis][11]
-
-: <br> - Code vulnerability violations <br> - Code quality violations
-
-[Software Composition Analysis][12]
-
-: <br> - Vulnerabilities <br> - Detected licenses
+| Source type     | Condition types |
+| --- | ----------- |
+| [**Static Code Analysis**][11] | <li> Code vulnerability violations <li> Code quality violations |
+| [**Software Composition Analysis**][12] | <li> Library vulnerability violations <li> Detected license violations |
+| [**Code Coverage**][15] | <li> Total code coverage threshold <li> Patch code coverage threshold |
+| [**Infrastructure as Code Scanning**][16] | <li> IaC vulnerability severity |
```

Comment:
Just noting that we recently changed the name to Infrastructure as Code Security. But it looks like the UI still refers to it as Infrastructure as Code Scanning. Could we pass this feedback along to the PR Gates PM?